docs/docs/start-here.mdx

```diff
@@ -0,0 +1,207 @@
+In fact, there are only two small files that provide Rasa what it needs. 
+
+The `data/flows/restaurants.yml` file defines the logic for this flow. In this example the logic is linear
+and walks the user through each of the steps in order. Each of the `question` steps fills the corresponding slot.
+
+```yaml
+flows:
+  recommend_restaurant:
+    description: This flow recommends a restaurant
+    steps:
+      - id: "0"
+        question: cuisine
+        skip_if_filled: true
+        next: "1"
+      - id: "1"
+        question: price_range
+        next: "2"
+      - id: "2"
+        question: part_of_town
+        next: "3"
+      - id: "3"
+        action: search_restaurants
+```
+
+To build more advanced flows, you can add conditional logic, link to other flows, and more. See [add link]
+
+In addition to the flows, the `domain.yml` file contains definitions of the slots and responses used in this flow. 
+
+### Understanding DM2
+
+Rasa uses a new approach to building AI assistants called DM2 (name TBD). 
+If you've built AI assistants before, you might look at your project and think that many things are missing.
+
+* There is no NLU data with intents and entities
+* There is no logic mapping an intent like "restaurant_search" to the start of the restaurant flow
+* There are no slot mappings
+* There is no explicit logic handling corrections, interruptions, or other unhappy paths.
+
+DM2 doesn't need any of these things to be able to handle the example conversations above. So, how does that work?
+
+Many developers are familiar with dialogue systems made up of separate NLU, dialogue, and NLG components
+(this is also how Rasa worked previously).
+
+In DM2, we have a different set of modules. 
+
+* The *conversation handling* component (name TBD) interprets the conversation so far and predicts 
+a series of commands to progress the state of the conversation.
+* The *business logic* component executes those commands and the logic of your flows. 
+
+The NLU systems you might be familiar with take a single user message as input, and aim to represent
+the meaning of that message by predicting intents and entities. 
+Instead, *conversation handling* considers the conversation as a whole (not just one message),
+and predicts the *intended effect* of the user's message. 
+
+As an example, let's look at using a yes/no question to fill a slot called `late_delivery`:
+
+<Chat caption="yes/no question">
+<ChatBotText>Has it been more than 10 business days since you placed your order?</ChatBotText>
+</Chat>
+
+
+When a user answers "yes" or "no", an NLU model predicts an intent like `affirm` or `deny`.
+A second step (usually handled by the dialogue manager) then maps the intents to
+the `True/False` values of the `late_delivery` slot. 
+
+Instead, the *conversation handling* component directly outputs the command `SetSlot("late_delivery", True)`.
```

Comment:
will we want to have a command reference with the initial release? I was thinking maybe it's not needed at first, hence we wouldn't necessary need to use the correct command syntax here. But instead we could say:

```suggestion
Instead, the *conversation handling* component directly outputs a command to set the `late_delivery` slot to `True`.
```

Reply:
(probably nitpicking but might be good for coherence)